### DISCUSSION TITLE
How to read the tecplot Binary file

### DISCUSSION THREAD

**Discussion by chenjiajun1991** [Kitware/trame#110] [Q&A]
I want to use tream to read the .plt data , but he told me that the encoding is wrong. After querying VTKTecopltReader, I can only read the ascii encoded .plt file, but my .plt file is a binary file. What can I do to read it with VTK and Render it.

**Comment by jourdain** [comment]
It seems that ParaView can read it via its [VisiBridgeReader](https://discourse.paraview.org/t/how-to-read-the-tecplot-binary-file-file/3538). Otherwise, you might be able to use [TecIO](https://www.tecplot.com/2021/06/23/back-to-basics-tecio/) to read it and it will be up to you to package it into a VTK data structure.
